tests/to-json-tests.js

```diff
@@ -7,7 +7,7 @@ test('can handle empty string', t => {
         type: "AdaptiveCard",
         body: [],
         actions: [],
-        version: "1.0"
+        version: "1.2"
```

Comment:
Should we make a version a constant so that anytime we do bump this up we only need to change it in one place for this file?

Reply:
@austinparkk sure i can do that - good point!